hw5/GameOfWords.java

```diff
@@ -0,0 +1,71 @@
+package org.hillel.hw5;
+
+import java.util.Objects;
+import java.util.Random;
+import java.util.Scanner;
+
+//Second version of Word Game
+public class GameOfWords {
+    //Main Array of vegetables and fruits
+    private final String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado",
```

Comment:
Please make the 'words' array a constant because its values don't seem to be changed.